content/blog/2024/frx/index.md

```diff
@@ -0,0 +1,64 @@
+
+We considered the landscape of tools and services, and felt that [FR-1] and [FR-2] were already well-served by a variety of tools and services for community workspace infrastructure (e.g., JupyterHub: https://jupyterhub.readthedocs.io), sharable computational environments (e.g., BinderHub: https://binderhub.readthedocs.io), authoring and reading computational narratives (e.g., Jupyter Book: https://jupyterbook.org and MyST: https://mystmd.org), and data I/O tools and standards (e.g., Zarr: https://zarr.readthedocs.io and Intake: https://intake.readthedocs.io). 
+
+However there was a natural missing piece for **[FR-3 Challenges]**, and we could not identify any community-managed infrastructure that facilitated data challenges. This is the goal of `frx-challenges`.
+
+## Why facilitate data challenges?
+
+Data challenges are harder than you think! While it is simple enough to run somebody else's code locally, data challenges require a systematic, secure, and automated approach to accepting and evaluating submissions in a fair and repeatable way. Here are some of the big challenges to tackle:
+
+- **Submissions must retain user and team identity**, which means that we must keep track of users and their submissions over time, since data challenges are designed to encourage iterative improvement and optimization.
+- **Evaluations must use potentially complex resources and data** since many data challenges operate by publicly sharing a small dataset, and then running it against a much more complex dataset.
+- **Evaluations must be totally secure**, so that submissions can't do nefarious things like mine cryptocurrency or gain access to the challenge's private data.
+- **Evaluations must be automated**, so that running the challenge does not require extensive human intervention and can scale to many users.
+- **Evaluation must be flexible**, so that the infrastructure can accept a variety of types of submissions (e.g. code, data, model weights, etc), run them with arbitrary environments designed by the organizers, and run them with the right hardware to get the job done.
+
+These are just a few of the major challenges that we've tried to address with `frx-challenges`, and we're excited to see how it goes with our first assisted community challenge: the [Cellmap Challenge](https://cellmapchallenge.janelia.org/).
+
+If you're interested in learning more or participating in this project, follow along at its GitHub repository:
+
+https://github.com/2i2c-org/frx-challenges
+
+This is still the **very early stages** of the project, and we imagine it will evolve significantly. We welcome feedback for how it can more effectively serve a variety of communities.
+
+## Acknowledgements
+
+Many thanks to the Howard Hughes Medical Institute (HHMI) for collaborating with us on the [Cellmap Challenge](https://cellmapchallenge.janelia.org/), which led to the creation of this project.
```

Comment:
I'm not sure what org Kristen Ratan belongs to, but let's add them to the acknowledgement as well.